src/pages/documentation/adobe-media-analytics/tabs/index.md

```diff
@@ -2,12 +2,6 @@
 
 The latest Android SDK versions:
 
-[<img src="https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/core.svg?logo=android&logoColor=white&label=core&style=flat-square" alt="Maven Central" style="width:150px"/>](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/core)
-
-[<img src="https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/analytics.svg?logo=android&logoColor=white&label=analytics&style=flat-square" alt="Maven Central" style="width:150px"/>](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/analytics)
-
-[<img src="https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/media.svg?logo=android&logoColor=white&label=media&style=flat-square" alt="Maven Central" style="width:150px"/>](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/media)
-
```

Comment:
Since we're removing this, we might need to change the value within repeat as well.

Reply:
Fixed.  I also remove the latest version links for iOS and JS.  We don't need to have them in ACP doc.